### PR TITLE
Create LICENSE.txt

### PR DESCRIPTION
**Pull request by kmcnellis**

### CODE REVIEW

**Comment by agundy** [comment]
Should be combined with the other license PR

**Comment by kmcnellis** [comment]
Does it matter? Done from the web interface.

**Comment by agundy** [comment]
Fine :P

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (seveibar) [inline LICENSE.txt]: we should really be GPL

**Pull request merged**